### ISSUE TITLE
Do not parse code

### ISSUE THREAD

**Issue by elfet** [thlorenz/doctoc#88]
Hi,

I have some code in my readme with &lt;h1&gt; tag:

```
'''html
<div>
  <h1>{{ title }}</h1>
  <p>
    {{ text }}
  </p>
</div>
'''
```
doctoc take `<h1>{{ title }}</h1>`  as title, but it's is code!

**Comment by pavestru** [comment]
+1 Almost the identical issue, including code!

**Comment by jez** [comment]
Closed via #107.
